java/gradle.java/src/org/netbeans/modules/gradle/java/api/GradleJavaSourceSet.java

```diff
@@ -42,12 +42,13 @@
         "LBL_JAVA=Java",
         "LBL_GROOVY=Groovy",
         "LBL_SCALA=Scala",
+        "LBL_KOTLIN=Kotlin",
         "LBL_RESOURCES=Resources",
         "LBL_GENERATED=Generated"
     })
     public static enum SourceType {
 
-        JAVA, GROOVY, SCALA, RESOURCES,
+        JAVA, GROOVY, SCALA, KOTLIN, RESOURCES,
```

Comment:
Always add enum at the end of the list when extending an API, aslo add javadoce with @since annotation. see GENERATED